### PR TITLE
Added sequential evaluation

### PR DESCRIPTION
**Pull request by mahdihamdani**

### CODE REVIEW

File: nex_cv/data_providers.py [hunk 1]
```diff
@@ -142,3 +143,162 @@ def __str__(self):
...
+        """
+
+        :param X: list(str): training sentences
+        :param y: list(dict): with intent_id and created (and updated) dates
+        :param date_key: key used for the date used in sorting the data
+        :param return_timestamps: if the timestamps are returned (e.g. used by the data provider)
+        :return: sorted dataset: X, y[, time_stamps]
+        """
+
+        time_stamps = [TimeSeqDatasetProvider.resolve_date(a, date_key) for a in y]
```
Comment (ksen0) [inline nex_cv/data_providers.py:213]: What if `resolve_date(..)` returns `None`? Should we just exclude that datapoint, conceptually?

**Review by ksen0 — APPROVED** [review]
Nice! Minor question about what happens when timestamp is None

**Comment by ksen0** [comment]
Please rebase and merge, rather than merging or squashing

File: nex_cv/data_providers.py (hunk 1 above)
Comment (mahdihamdani) [reply nex_cv/data_providers.py:213]: The None values are filtered in `sort_dataset_by_date`

**Pull request merged**